app/upgrades/v0.0.11.go

```diff
@@ -0,0 +1,99 @@
+package upgrades
+
+import (
+	"context"
+
+	storetypes "cosmossdk.io/store/types"
+	upgradetypes "cosmossdk.io/x/upgrade/types"
+	cosmosTypes "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	"github.com/pokt-network/poktroll/app/keepers"
+	sessiontypes "github.com/pokt-network/poktroll/x/session/types"
+	tokenomicstypes "github.com/pokt-network/poktroll/x/tokenomics/types"
+)
+
+// Upgrade_0_0_11 is the upgrade handler for v0.0.11 Alpha TestNet upgrade
+// Beta TestNet was launched with v0.0.11, so this upgrade is exclusively for Alpha TestNet.
+//   - Before: v0.0.10
+//   - After: v0.0.11
+var Upgrade_0_0_11 = Upgrade{
+	PlanName: "v0.0.11",
+	CreateUpgradeHandler: func(mm *module.Manager,
+		keepers *keepers.Keepers,
+		configurator module.Configurator,
+	) upgradetypes.UpgradeHandler {
+
+		// Adds new parameters using ignite's config.yml as a reference. Assuming we don't need any other parameters.
+		// https://github.com/pokt-network/poktroll/compare/v0.0.10...v0.0.11-rc
+		applyNewParameters := func(ctx context.Context) (err error) {
+			logger := cosmosTypes.UnwrapSDKContext(ctx).Logger()
+			logger.Info("Starting parameter updates for v0.0.11")
+
+			// Set num_suppliers_per_session to 15
+			// Validate with: `poktrolld q session params --node=https://testnet-validated-validator-rpc.poktroll.com/`
```

Comment:
Is this the correct URL and value has been validated?

Reply:
Yes and yes.